internal/fs/inode/file_mock_bucket_test.go

```diff
@@ -152,3 +155,63 @@ func (t *FileMockBucketTest) TestFlushSyncedFileForceFetchObjectFromGCS() {
 	require.NoError(t.T(), err)
 	t.bucket.AssertExpectations(t.T())
 }
+
+func (t *FileMockBucketTest) TestSync_RemoteAppendClobber() {
+	// Expect CreateObject from createLockedInode
+	t.bucket.On("CreateObject", t.ctx, mock.AnythingOfType("*gcs.CreateObjectRequest")).
+		Return(&gcs.Object{Name: fileName, Generation: 1, MetaGeneration: 1}, nil)
+
+	t.createLockedInode(fileName, emptyGCSFile)
+	assert.False(t.T(), t.in.IsLocal())
+
+	// Dirty the file.
+	_, err := t.in.Write(t.ctx, []byte("data"), 0, util.NewOpenMode(util.WriteOnly, 0))
```

Comment:
please arrange these tests in the following form
arrange
--new line
act
--new line
assert